compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -378,4 +379,14 @@ class ComposePanel @ExperimentalComposeUiApi constructor(
     fun renderImmediately() {
         _composeContainer?.renderImmediately()
     }
+
+    @ExperimentalComposeUiApi
```

Comment:
```suggestion
    @InternalComposeUiApi
```

Discussed with Ivan, better to mark it fully as internal, as we don't have a a robust semantics for this field for public use (it also, means we need to remove `@InternalCoreApi` from the setter)